packages/slice-machine/src/legacy/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -79,6 +81,11 @@ const FieldZones: FC = () => {
     setIsDeleteRepeatableZoneDialogOpen,
   ] = useState(false);
 
+  const tableFieldExperiment = useTableFieldExperiment();
+  const maybeFilteredItemsWidgetsArray = tableFieldExperiment.eligible
```

Comment:
sholud we also take into account this for `primaryWidgetsArray`? probably a good idea to do the filtering directly when we declare the array, to avoid any issues

Reply:
Good catch, I fixed it.